--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -83557,6 +83557,7 @@
   let n = jB(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (UE({
+    enabled: !1,
     dsn: SU,
     environment: e.buildFlavor,
     release: CU(n.version),
--- a/.vite/build/window-all-closed-BwlaNiSa.js
+++ b/.vite/build/window-all-closed-BwlaNiSa.js
@@ -40848,6 +40848,7 @@
     i = t.da(e.version);
   return (
     Hz({
+      enabled: !1,
       dsn: t.ua,
       environment: YB.buildFlavor,
       release: i,
