--- a/webview/assets/app-initial-BTphDPeq.js
+++ b/webview/assets/app-initial-BTphDPeq.js
@@ -236473,7 +236473,7 @@
       );
   return (
     o.forEach((r) => {
-      if (e?.has(r.model) === !0 || (u ? n.has(r.model) : !r.hidden)) {
+      if (e?.has(r.model) === !0 || !r.hidden) {
         let e = a
             ? r.supportedReasoningEfforts
             : r.supportedReasoningEfforts.filter(
@@ -236704,13 +236704,13 @@
   userSavedReasoningEffort: t,
   listModelsData: n,
 }) {
-  let r = e ? GM(n?.models, e) : (n?.defaultModel ?? GM(n?.models, `gpt-5.5`)),
+  let r = e ? GM(n?.models, e) : (n?.defaultModel ?? n?.models[0] ?? null),
     i = r?.supportedReasoningEfforts?.map((e) => e.reasoningEffort),
     a = t != null && i != null && i.includes(t) ? t : r?.defaultReasoningEffort;
   return {
-    model: r ? r.model : (e ?? `gpt-5.5`),
+    model: r?.model ?? e ?? null,
     reasoningEffort:
-      a ?? t ?? n?.defaultModel?.defaultReasoningEffort ?? `medium`,
+      a ?? t ?? n?.defaultModel?.defaultReasoningEffort ?? null,
     profile: null,
     isLoading: !1,
   };
