--- a/webview/assets/app-initial-BTphDPeq.js
+++ b/webview/assets/app-initial-BTphDPeq.js
@@ -181872,11 +181872,16 @@
   unstable_useTransitions: i,
 }) {
   let a = Gw.useRef();
-  a.current ??= kzn({ initialEntries: n, initialIndex: r, v5Compat: !0 });
+  a.current ??= kzn({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: r,
+    v5Compat: !0,
+  });
   let o = a.current,
     [s, c] = Gw.useState({ action: o.action, location: o.location }),
     l = Gw.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         i === !1 ? c(e) : Gw.startTransition(() => c(e));
       },
       [i],
@@ -197825,7 +197830,7 @@
       (gar = `app-shell-bottom-panel-launcher-visible`),
       (_ar = `app-shell-file-tree-open`),
       (yar = 100),
-      (fO = Da(Q, !0)),
+      (fO = Da(Q, window.__ELECTRON_SHIM__.initialSidebarState)),
       (bar = Da(Q, !0)),
       (pO = Da(Q, !1)),
       (xar = Da(Q, null)),
@@ -197869,7 +197874,10 @@
           };
         },
       })),
-      (war = Da(Q, () => new qRe(1))),
+      (war = Da(
+        Q,
+        () => new qRe(window.__ELECTRON_SHIM__.initialSidebarState),
+      )),
       (Tar = Da(Q, qir)),
       (Ear = Hm(gar, !0)),
       (mO = Da(ST, !1)),
